tools/fs/files.ts

```diff
@@ -215,16 +215,6 @@ export function getToolsVersion() {
       parsed = JSON.parse(readFile(isopackJsonPath))["isopack-1"];
       return parsed.name + '@' + parsed.version;
     }
-
-    // XXX COMPAT WITH 0.9.3
-    const unipackageJsonPath = pathJoin(
-      getCurrentToolsDir(),
-      '..',  // get out of tool, back to package
-      'unipackage.json'
-    );
-    parsed = JSON.parse(readFile(unipackageJsonPath));
-    return parsed.name + '@' + parsed.version;
-
```

Comment:
It should get replaced with an error, right? I mean, _if_ this branch is still reachable, this function will return `undefined`.

Reply:
Adding

Reply:
Decided to change the attack vector. Momentarily returning the functionality.